docs/source/apidocs/gfwapiclient/gfwapiclient.__version__.rst

```diff
@@ -25,6 +25,6 @@ API
 
 .. py:data:: __version__
    :canonical: gfwapiclient.__version__.__version__
-   :value: '0.1.0'
+   :value: '1.0.1'
```

Comment:
I think this should be 1.0.2, since we deployed the first version as 1.0.1. We had to do that because a test version "1.0.0" was deployed before in production PYPI (instead of the test PYPI server) and PYPI don't let you reuse the same version numbers previously used.

Reply:
Sure. It will be bumped automatically with [cz bump](https://commitizen-tools.github.io/commitizen/getting_started/#bump-version) on the release when we rebuild the docs: https://github.com/GlobalFishingWatch/gfw-api-python-client/blob/develop/pyproject.toml#L234